nautobot/dcim/filters/mixins.py

```diff
@@ -194,10 +194,31 @@ class DeviceModuleCommonFiltersMixin(django_filters.FilterSet):
         field_name="module_bays",
         label="Has module bays",
     )
+    has_empty_module_bays = django_filters.BooleanFilter(
+        method="filter_has_empty_module_bays",
+        label="Has empty module bays",
+    )
     module_bays = django_filters.ModelMultipleChoiceFilter(
```

Comment:
Might be appropriate here? I was thinking the module bay "position" wouldn't be suitable for a natural key since in the common case they're going to be integers.